test/core/end2end/fuzzers/api_fuzzer.cc

```diff
@@ -260,18 +259,14 @@ static void sched_connect(grpc_closure* closure, grpc_endpoint** ep,
                           gpr_timespec deadline);
 
 typedef struct {
-  grpc_timer timer;
   grpc_closure* closure;
   grpc_endpoint** ep;
   gpr_timespec deadline;
 } future_connect;
 
-static void do_connect(void* arg, grpc_error_handle error) {
+static void do_connect(void* arg) {
```

Comment:
This also does not need to be a `void*` anymore.

Reply:
Done.